python/test/reprocess_all_met.py

```diff
@@ -71,7 +71,7 @@ def submit_download_job_for_date(date):
 
     # submit the job
     batch = boto3.client('batch')
-    latest = get_latest_revision('ios_ingest_met_job')
+    latest = get_latest_revision('ios_ingest_nrl_job')
     batch.submit_job(
         jobName='met_%s' % date,
         jobDefinition='ios_ingest_met_job:%d' % latest,
```

Comment:
This needs to change too.

Maybe create a variable, jobDefinitionName = 'ios_ingest_nrl_job' and use it in those four places.

Reply:
Thanks, @hahnd.